srcpkgs/github-cli/template

```diff
@@ -22,6 +23,19 @@ pre_build() {
 	fi
 }
 
+post_build() {
+	gopath=$(go env GOPATH)
+	gh=$(find $gopath/bin -name gh)
```

Comment:
We set `GOPATH` for the build, so it should work just as well with `find $GOPATH/bin`